sleap_io/model/labeled_frame.py

```diff
@@ -59,5 +59,5 @@ def numpy(self) -> np.ndarray:
         n_nodes = len(self.instances[0]) if n_instances > 0 else 0
         pts = np.full((n_instances, n_nodes, 2), np.nan)
         for i, inst in enumerate(self.instances):
-            pts[i] = inst.numpy()
+            pts[i] = inst.numpy()[:, 0:2]
         return pts
```

Comment:
Modified s.t. `labeled_frame.numpy` returns points only even if there are `PredictedInstances`. The other option is to make `labeled_frame.numpy` always return 3 coordinates with the last coordinate (the score) equal to `np.nan` if type is `Instance`. The current solution (no score returned) seems cleaner and more consistent to me personally.